### PR TITLE
feat: add custom title to storybook

### PR DESCRIPTION
**Pull request by feedm3**
Just a little bit of customization. The best name I could imagine was "Satellytes UI", feel free to propose something else.

I first checked how WFP is doing it: They have a custom React component to display different styles and a dynamic version number with a dropdown. They then have a somehow hacky webpack call to replace the official logo component with theirs: https://github.com/wfp/designsystem/blob/master/.storybook/main.js#L39-L45

After fiddling out with that I decided to Google the problem, as it felt too hacky to change a simple title. I then found out that there is a theming object where you can just set the title 😄

### CODE REVIEW

File: .storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { create } from '@storybook/theming';
+
+export default create({
```
Comment (fischeversenker) [inline .storybook/theme.js]: I'm not a fan of default exports. Does this need to be one?

**Review by fischeversenker — APPROVED** [review]
Nice that you figured this out 🙌 
Regarding the name: I can't think of anything that would be more fitting right now, also I do like "Satellytes UI" so I'm perfectly fine with it. 
One question regarding the `default` theme export, please see my comment.
It does not block this PR so I'm approving.

File: .storybook/theme.js (hunk 1 above)
Comment (feedm3) [reply .storybook/theme.js]: Didn't even think of it as those configuration files usually rely on the default export. But this shouldn't be the case for this file. I change it to `export const customTheme = create(...)`. Let's wait for the build if it still works. If yes, I will merge.

Comment (feedm3) [reply .storybook/theme.js]: It works, will merge it. Thanks.

**Pull request merged**